safe-room.lic

```diff
@@ -26,9 +26,22 @@ class SafeRoom
 
     return unless args.force || need_healing?
 
-    if use_pc_empath?(settings.safe_room_id, settings.safe_room_empath)
-      ensure_copper_on_hand(settings.safe_room_tip_threshold || 0, settings)
-      tip(settings.safe_room_tip_threshold, settings.safe_room_tip_amount, settings.safe_room_empath)
+    pc_empaths = settings.safe_room_empaths || []
```

Comment:
What about for people that want to use UserVars to store their empath list for the sake of privacy?

Reply:
Is this line now unnecessary?

Reply:
Yes - thank you base-empty.

Reply:
Guessing you mean the `|| []`

Reply:
We still need to initialize pc_empaths